src/js/hca-rjsf/config/form.js

```diff
@@ -24,6 +46,91 @@ const formConfig = {
           schema: {}
         }
       }
+    },
+    militaryService: {
+      title: 'Military service',
+      pages: {
+        serviceInformation: {
+          path: 'military-service/service-information',
+          title: 'Service periods',
+          uiSchema: {
+            lastServiceBranch: {
+              'ui:title': 'Last branch of service',
+              'ui:options': {
+                labels: lastServiceBranchLabels
+              }
+            },
+            // TODO: this should really be a dateRange, but that requires a backend schema change. For now
+            // leaving them as dates, but should change these to get the proper dateRange validation
+            lastEntryDate: currentOrPastDate.uiSchema('Start of service period'),
```

Comment:
I think this should be changed to a `dateRange` in the backend, but not sure how to ensure nothing breaks with the live application. I neglected to set up time to discuss that, but will bring it up on #rainbows-team to see who wants to join a quick chat on how to orchestrate any backend changes.